packages/vc.js/src/vc-ld/__tests__/relative-refs/__fixtures__/documentLoader.ts

```diff
@@ -1,7 +1,7 @@
 import { contexts } from "./contexts";
 import controller from "./controller.json";
 
-// import jsonld from "jsonld";
+// import jsonld from "@transmute/jsonld";
```

Comment:
Remove comments if we are not using it here, how come we don't use the library in this spot anymore?